pkg/destroy/gcp/gcp.go

```diff
@@ -196,6 +196,10 @@ func (o *ClusterUninstaller) clusterIDFilter() string {
 	return fmt.Sprintf("name eq \"%s-.*\"", o.ClusterID)
 }
 
+func (o *ClusterUninstaller) clusterLabelFilter() string {
+	return fmt.Sprintf("labels.kubernetes-io-cluster-%s eq \"owned\"", o.ClusterID)
```

Comment:
would love if you include pointer to doc for filter on labels in the commit message.